src/aiortc/jitterbuffer.py

```diff
@@ -26,22 +29,29 @@ def capacity(self) -> int:
     def add(self, packet: RtpPacket) -> Optional[JitterFrame]:
         if self._origin is None:
             self._origin = packet.sequence_number
-        elif packet.sequence_number <= self._origin - MAX_MISORDER:
-            self.remove(self.capacity)
-            self._origin = packet.sequence_number
-        elif packet.sequence_number < self._origin:
-            return None
-
-        delta = packet.sequence_number - self._origin
-        if delta >= 2 * self.capacity:
-            # received packet is so far beyond capacity we cannot keep any
-            # previous packets, so reset the buffer
-            self.remove(self.capacity)
-            self._origin = packet.sequence_number
-        elif delta >= self.capacity:
-            # remove just enough packets to fit the received packets
+            delta = 0
+            misorder = 0
+        else:
+            delta = (packet.sequence_number - self._origin) % self.__max_number
```

Comment:
Can we use uint16_add (defined in aiortc.utils) for consistency?

Reply:
I don't think so - I believe in python (-2 & 0xFFFF) = 2 while  (-2 % 65536) = 65534

Reply:
Nope they are identical:

```>>> -2 & 0xFFFF
65534
>>> -2 % 65536
65534```

It's good news too, otherwise all our sequence number calculations for what we send would be messed up..